package.json

```diff
@@ -26,12 +26,30 @@
   },
   "homepage": "https://github.com/vacuumlabs/webpack-config-vacuumlabs#readme",
   "dependencies": {
```

Comment:
I am not sure, if it's ok to move these to direct dependencies. But this is a topic for longer discussion.